conversions/src/types.rs

```diff
@@ -165,6 +165,10 @@ impl Signature {
             s: Secp256K1Scalar::from_noncanonical_biguint(BigUint::from_bytes_be(&s)),
         }
     }
+
+    pub fn sign_message(private_key: &PrivateKey, message: &Message) -> Self {
```

Comment:
should `sign_message` not be a method on `PrivateKey` so that we don't need to expose the private key outside its type?

Reply:
Good catch, changed it accordingly